test/workbook/worksheet/tc_cell.rb

```diff
@@ -319,6 +319,21 @@ def test_to_xml_string_array_formula
     assert(doc.xpath("//f[@ref='A1']"))
```

Comment:
@randym also, this test seems to be false-positive. `doc.xpath(...)` always returns non-falsey value. It returns empty array if node is not found. That's why I've decided to use `empty?` and `any?` in my test.